client/login/wp-login/style.scss

```diff
@@ -36,6 +36,20 @@ $image-height: 47px;
 			display: none;
 		}
 	}
+
+	.step-container-v2__heading {
```

Comment:
@chriskmnds, can you please create a follow-up PR that removes this modification?

If you always want to centralize the heading and subheading in the login page, you can pass the `align` prop to `<Step.Heading />` as `center`.

Reply:
That sounds nice. I'll make a note and happy to address in a follow-up. 

I am not yet sure on the exact shape of the final structure here either, so some things are still intermediary.